android/RNKeys.gradle

```diff
@@ -61,62 +61,66 @@ def loadKeys(){
       println("\n\n")
       if (project.ext.has('keyFiles') && project.ext.keyFiles.size() > 0) {
         keysFile = project.ext.keyFiles[project.ext.keyFiles.keySet().iterator().next()];
-        exportCommand="export KEYSFILE="+keysFile+" && ";
-        println "Choose "+keysFile+" file for react-native-keys to avoid stop any build process\n\n"
+        exportCommand = "export KEYSFILE=" + keysFile + " && ";
+        println "Choose " + keysFile + " file for react-native-keys to avoid stop any build process\n\n"
       } else {
         // Print an error message
         println "Please define project.ext.keyFiles as you are using react-native-keys\n\n"
         return;
       }
     }
 
-    def IS_EXAMPLE=safeExtGet("IS_EXAMPLE",false);
+    def IS_EXAMPLE = safeExtGet("IS_EXAMPLE", false);
 
-    if(IS_EXAMPLE)
-    {
-      exportCommand=exportCommand+"export IS_EXAMPLE='TRUE' && ../../keysAndroid.js"
-    }
-    else
-    {
+    if (IS_EXAMPLE) {
+      exportCommand = exportCommand + "export IS_EXAMPLE='TRUE' && ../../keysAndroid.js"
+    } else {
       def nodeModulesDir = getNodeModulesDir()
-      exportCommand=exportCommand+nodeModulesDir+"/node_modules/react-native-keys/keysAndroid.js"
+      exportCommand = exportCommand + nodeModulesDir + "/node_modules/react-native-keys/keysAndroid.js"
     }
-    def proc =["/bin/sh", "-c", exportCommand].execute()
+    def proc = ["/bin/sh", "-c", exportCommand].execute()
     proc.consumeProcessOutput(System.out, System.err)
 
     println(keysFile)
-    def keys = [:]
+    def keys = [: ]
     File f = new File("$project.rootDir/../$keysFile");
 
     if (f.exists()) {
       def parsedJson = new groovy.json.JsonSlurper().parseText(f.text)
-      parsedJson.public.each { key, value ->
+      parsedJson.public.each {
+        key,
+        value ->
         keys.put(key, value.replace('"', '\\"'))
       }
     } else {
       println("**************************")
       println("*** Missing KEYs file ****")
       println("**************************")
-      return ;
+      return;
     }
 
     project.ext.set("keys", keys)
 
     android {
       defaultConfig {
-        keys.each { k, v ->
-          def escaped = v.replaceAll("%","\\\\u0025")
-          buildConfigField "String", k, "\"$v\""
-          resValue "string", k, "\"$escaped\""
+        keys.each {
```

Comment:
@MasonLe2497 can you reverse only this formatting on because it is unable to readable

Reply:
Yep,  i updated my code. thanks